github.sgnl.yaml

```
@@ -0,0 +1,700 @@
+        externalId: mergeCommitAllowed
+        type: Bool
+      - name: name
+        externalId: name
+        type: String
+      - name: nameWithOwner
+        externalId: nameWithOwner
+        type: String
+      - name: projectsResourcePath
+        externalId: projectsResourcePath
+        type: String
+      - name: pushedAt
+        externalId: pushedAt
+        type: DateTime
+      - name: rebaseMergeAllowed
+        externalId: rebaseMergeAllowed
+        type: Bool
+      - name: resourcePath
+        externalId: resourcePath
+        type: String
+      - name: squashMergeAllowed
+        externalId: squashMergeAllowed
+        type: Bool
+      - name: stargazerCount
+        externalId: stargazerCount
+        type: Int64
+      - name: tempCloneToken
+        externalId: tempCloneToken
+        type: String
+      - name: updatedAt
+        externalId: updatedAt
+        type: DateTime
+      - name: usesCustomOpenGraphImage
+        externalId: usesCustomOpenGraphImage
+        type: Bool
+      - name: viewerCanAdminister
+        externalId: viewerCanAdminister
+        type: Bool
+      - name: viewerCanCreateProjects
+        externalId: viewerCanCreateProjects
+        type: Bool
+      - name: viewerCanSubscribe
+        externalId: viewerCanSubscribe
+        type: Bool
+      - name: viewerCanUpdateTopics
+        externalId: viewerCanUpdateTopics
+        type: Bool
+      - name: viewerDefaultCommitEmail
+        externalId: viewerDefaultCommitEmail
+        type: String
+      - name: viewerHasStarred
+        externalId: viewerHasStarred
+        type: Bool
+      - name: viewerPossibleCommitEmails
+        externalId: viewerPossibleCommitEmails
+        type: String
+        list: true
+      - name: webCommitSignoffRequired
+        externalId: webCommitSignoffRequired
+        type: Bool
+      # Note: This attribute has been added in post-processing and is not part of the Team entity.
+      - name: enterpriseId
+        externalId: enterpriseId
+        type: String
+      # Note: This attribute has been added in post-processing and is not part of the Team entity.
+      - name: orgId
+        externalId: orgId
+        type: String
+        indexed: true
+        attributeAlias: Repository-orgId
+  RepositoryCollaborator:
+    parent: Repository
+    displayName: RepositoryCollaborator
+    externalId: $.collaborators.edges
+    description: Entity representing the metadata of a User (Collaborator) entity assigned to a GitHub Repository
+    pageSize: 100
+    pagesOrderedById: false 
+    attributes:
+      - name: collaboratorId
+        externalId: $.node.id
+        type: String
+        indexed: true
+        uniqueId: true
+        attributeAlias: repositoryCollaboratorId
+      - name: permission
+        externalId: permission
+        type: String
+  # Collaborator is a User entity type: https://docs.github.com/en/graphql/reference/objects#repositorycollaboratoredge
+  Collaborator:
```

```suggestion

  # Collaborator is a User entity type: https://docs.github.com/en/graphql/reference/objects#repositorycollaboratoredge
  Collaborator:
```
nit: Add new line between all entities

---

None of the other templates have new lines between entities so I'll keep that format unless you want to change this